delta/src/main/scala/ch/epfl/bluebrain/nexus/delta/RepairFromMessages.scala

```diff
@@ -18,6 +25,18 @@ object RepairFromMessages {
 
   private val log = Logger[RepairFromMessages.type]
 
+  private val parallelism = sys.env.getOrElse("REPAIR_FROM_MESSAGES_PARALLELISM", "1").toIntOption.getOrElse(1)
```

Comment:
You could have use a property from `migration.conf` (MigrationConfig in AppConfig), since this already exists and it is used for this purpose.